setup.sh

```diff
@@ -97,72 +97,104 @@ function _inspect
+    does not work with all scripts.
+
+\e[38;5;214mOPTIONS\e[39m
+    \e[94mConfig path, container or image adjustments\e[39m
+        -i IMAGE_NAME
+            Provides the name of the docker-mailserver image. The default value is
+            \e[37mdocker.io/mailserver/docker-mailserver:latest\e[39m
+
+        -c CONTAINER_NAME
+            Provides the name of the running container.
+
+        -p PATH
+            Provides the config folder path. The default is
+            \e[37m${CDIR}/config/\e[39m
+    \e[94mSELinux\e[39m
+        -z
+            Allows container access to the bind mount content that is shared among
+            multiple containers on a SELinux-enabled host.
+
+        -Z
+            Allows container access to the bind mount content that is private and
+            unshared with other containers on a SELinux-enabled host.
+
+    \e[94mOthers\e[39m
+        -h \e[31m|\e[39m help
+            Shows this help dialogue.
+
+\e[38;5;214mCOMMAND\e[39m
+    \e[94mOBJECT\e[39m email \e[31m:=\e[39m
+        ${0} email add <EMAIL ADDRESS> [<PASSWORD>]
+        ${0} email update <EMAIL ADDRESS> [<PASSWORD>]
+        ${0} email del [ ARGUMENTS\e[31m...\e[39m ] <EMAIL ADDRESS>
+        ${0} email restrict <add\e[31m|\e[39mdel\e[31m|\e[39mlist> <send\e[31m|\e[39mreceive> [<EMAIL ADDRESS>]
+        ${0} email list
+
+    \e[94mOBJECT\e[39m alias \e[31m:=\e[39m
+        ${0} alias add <EMAIL ADDRESS> <RECIPIENT>
+        ${0} alias del <EMAIL ADDRESS> <RECIPIENT>
+        ${0} alias list
+
+    \e[94mOBJECT\e[39m quota \e[31m:=\e[39m
+        ${0} quota set <EMAIL ADDRESS> [<QUOTA>]
+        ${0} quota del <EMAIL ADDRESS>
+
+    \e[94mOBJECT\e[39m config \e[31m:=\e[39m
+        ${0} config dkim [ ARGUMENTS\e[31m...\e[39m ]
+        ${0} config ssl <FQDN> (\e[96mATTENTION\e[39m: This is deprecated and will be removed soon.)
+
+    \e[94mOBJECT\e[39m relay \e[31m:=\e[39m
+        ${0} relay add-domain <DOMAIN> <HOST> [<PORT>]
+        ${0} relay add-auth <DOMAIN> <USERNAME> [<PASSWORD>]
+        ${0} relay exclude-domain <DOMAIN>
+
+    \e[94mOBJECT\e[39m debug \e[31m:=\e[39m
+        ${0} debug fetchmail
+        ${0} debug fail2ban [unban <IP>]
+        ${0} debug show-mail-logs
+        ${0} debug inspect
+        ${0} debug login <COMMANDS>
+
+\e[38;5;214mEXAMPLES\e[39m
+    \e[37m./setup.sh email add test@domain.tld\e[39m
+        Add the email account \e[37mtest@domain.tld\e[39m. You will be prompted to input a
+        password afterwards since no password was supplied.
+
+    \e[37m./setup.sh config dkim size 2048 selector ds\e[39m
+        Creates keys of length 2048 bit in a default setup where domains are obtained from
+        your accounts. The DKIM selector used is 'ds'.
+
+    \e[37m./setup.sh config dkim help\e[39m
+        This will provide you with a detailed explanation on how to use the \e[37mconfig
+        dkim\e[39m command, showing what arguments can be passed and what they do.
```

Comment:
I really appreciate the examples section 👍🏻 
Although dkim is using a selector which is not (yet) implemented

Reply:
I've changed that to use the `domain ...` argument :D